### PR TITLE
EPP to set method document

### PR DESCRIPTION
**Pull request by mayabrandi**
### Added
- EPP to set method document

### Review:
- [ ] Code approved by
- [x] Tests executed on stage by @mayabrandi 
- [ ] "Merge and deploy" approved by

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by mayabrandi** [comment]
Tested on stage together with [this pr.](https://github.com/Clinical-Genomics/servers/pull/716)
Skärmavbild 2021-12-16 kl  13 18 33

File: cg_lims/EPPs/udf/set/set_method.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+def set_methods_and_version(document_udfs: List[str], process: Process, host: str) -> None:
+    """Set Method Documents and Atlas Version"""
+
+    method_documents = get_document_paths(document_udfs=document_udfs, host=host, process=process)
+    if not method_documents:
+        raise MissingUDFsError(message="Found no valid Method document UDFs in the step.")
+    version_response = requests.get(f"{host}/version")
+    if version_response.status_code != 200:
```
Comment (Mropat) [inline cg_lims/EPPs/udf/set/set_method.py:49]: if we handle 404 and 403 differently already in get_path, then maybe we should do that always?

File: cg_lims/exceptions.py [hunk 2]
```diff
@@ -10,6 +10,12 @@ def __init__(self, message):
         self.message = message
 
 
+class AtlasResponseFailedError(LimsError):
+    """Raise when failing to route artifacts."""
```
Comment (Mropat) [inline cg_lims/exceptions.py:14]: update error docstring

File: cg_lims/EPPs/udf/set/set_method.py (hunk 1 above)
Comment (mayabrandi) [reply cg_lims/EPPs/udf/set/set_method.py:49]: yes

**Pull request merged**